rules-tests/Php80/Rector/Switch_/ChangeSwitchToMatchRector/Fixture/skip_unrelated_return.php.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\Switch_\ChangeSwitchToMatchRector\Fixture;
+
+final class UnrelatedReturn
+{
+    public function run($data, $proxyQuery, $alias): bool
+    {
+        switch ($data->getValue()) {
+            case ProductService::SPECIAL_GESTION_CAPSULE:
+                $proxyQuery->getQueryBuilder()->andWhere($alias . '.collectionCapsule = true');
+                break;
+            case ProductService::SPECIAL_GESTION_PRESTIGE:
+                $proxyQuery->getQueryBuilder()->andWhere($alias . '.editionPrestige = true');
+                break;
+        }
+
+        return true;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php80\Rector\Switch_\ChangeSwitchToMatchRector\Fixture;
+
+final class UnrelatedReturn
+{
+    public function run($data, $proxyQuery, $alias): bool
+    {
+        match ($data->getValue()) {
```

Comment:
@samsonasik something is wrong with this test. either its not a skip-test (and the filename is wrong), or the expectations are wrong